pkg/api/v1/types.go

```diff
@@ -3404,6 +3404,15 @@ type PodExecOptions struct {
 	Command []string `json:"command" protobuf:"bytes,6,rep,name=command"`
 }
 
+// PodPortForwardOptions is the query options to a Pod's port forward call.
+type PodPortForwardOptions struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// List of ports to forward
```

Comment:
Expand the godoc for this field and for the type that a consumer using them would have some indication of what they are doing and how it is used.  You should comment that this field is not used when the SPDY protocol is invoked in protocol versions up to the current one (the docs for port forward should be declaring a protocol number, if not, say that the first SPDY protocol version does not support port forward).